<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSearchGUI</class>
 <widget class="QWidget" name="ProjectSearchGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>609</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Text</string>
     </property>
     <property name="buddy">
      <cstring>searchFor</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchFor">
     <property name="placeholderText">
      <string>Search for</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Where to search</string>
     </property>
     <property name="buddy">
      <cstring>sourceCombo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="sourceCombo"/>
   </item>
   <item>
    <widget class="QLabel" name="pathEditLabel">
     <property name="text">
      <string>&amp;Path</string>
     </property>
     <property name="buddy">
      <cstring>pathEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PathWidget" name="pathEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="searchInBinaryFiles">
     <property name="text">
      <string>Search in &amp;binary files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>&amp;Search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="LoadingWidget" name="progressIndicator" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Include files:</string>
     </property>
     <property name="buddy">
      <cstring>includeFiles</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="includeFiles">
     <property name="placeholderText">
      <string>Files to include</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>E&amp;xclude files:</string>
     </property>
     <property name="buddy">
      <cstring>excludeFiles</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="excludeFiles">
     <property name="placeholderText">
      <string>Files to exclude</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathWidget</class>
   <extends>QLineEdit</extends>
   <header>pathwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoadingWidget</class>
   <extends>QWidget</extends>
   <header>widgets/LoadingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>searchFor</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>includeFiles</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeFiles</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
